bin/system-integrity

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python
+
+# system-integrity
```

Comment:
May I suggest to call this guy `kano-system-integrity` or similar to know it comes from us?

Reply:
Moving forward, I think we need to move away from kano-ifying everything and build things that solve problems not just for us.